apps/web/src/server/views/applications/representations/representation-details/representation-details.njk

```diff
@@ -0,0 +1,398 @@
+{% extends "applications/layouts/applications.layout.njk" %}
```

Comment:
make some grouped components out of this to break it down.

Reply:
-break down *my* component